daemon/main.c

```diff
@@ -1488,6 +1493,12 @@ int main(int argc, char **argv) {
         else debug(D_SYSTEM, "Not starting thread %s.", st->name);
     }
 
+    // ------------------------------------------------------------------------
+    // Start logsmanagement
+#ifdef ENABLE_LOGSMANAGEMENT
+    logsmanagement_main();
```

Comment:
`logsmanagement_main()` calls `uv_run(uv_default_loop(), UV_RUN_DEFAULT);` at the end which is blocking while there are events to process. Shouldn't then `logsmanagement_main()` be called in a separate thread?